tests/backend_test.py

```diff
@@ -857,8 +857,8 @@ def test_ibmq_emulator(
 
             c_cop_2 = c.copy()
             c_cop_2 = b_aer.get_compiled_circuit(c_cop_2, ol)
-            if ol == 0:
-                assert not all(pred.verify(c_cop_2) for pred in b.required_predicates)
+            # if ol == 0:
```

Comment:
The circuit is already fully supported by the simulator without compilation, so this test fails with the change